tests/js/destructuring-types/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -60,7 +60,7 @@ const Component3 = (props: {
 
 exports[`prop-types.js format 2`] = `
 ====================================options=====================================
-parsers: ["typescript"]
+parsers: ["babel-ts", "typescript"]
```

Comment:
No need to do this, it's already [done automatically](https://github.com/prettier/prettier/blob/f3343504f732b0570153da83d2d4eaee8de2d9d6/tests_config/run_spec.js#L157).

Reply:
I know. I am actually doing this as part of a workaround. I would like to run `tests/js/destructuring-types` across default Babel, Flow, and TypeScript parsers, but the output is a little different when using the TypeScript parsers and ran into an issue with the allowed parsers. I would definitely be interested if you can think of a better solution.

I will now move the new tests out of `tests/js/destructuring/destructuring.js`. Thanks for your quick and helpful reviews.

Reply:
The test scripts don't support such a situation. It's implied that either the output is the same for the two parser or that one of them throws an error. And I'm quite sure it should stay this way. If the output is different, it has to be fixed. Did you find a difference between `babel-ts` and `typescript` in the stable version?

Reply:
By "TypeScript parsers" I meant both `babel-ts` and `typescript`. In this case both TypeScript parsers will output a semicolon (`;`) while the other parsers would output a comma (`,`). Using separate `run_spec` statements seems to solve this for me.